src/init/ext_manifest.c

```diff
@@ -70,3 +70,18 @@ const struct ext_man_probe_support ext_man_probe
 #endif
 	},
 };
+
+const struct ext_man_config_data ext_man_config
+	__aligned(EXT_MAN_ALIGN) __section(".fw_metadata") = {
+	.hdr.type = EXT_MAN_ELEM_CONFIG_DATA,
+	.hdr.elem_size = ALIGN_UP(sizeof(struct ext_man_config_data),
+				  EXT_MAN_ALIGN),
+	.elems = {
+#if CONFIG_CAVS_LPRO
+		{
+			.token = EXT_MAN_CONFIG_CAVS_LPRO,
+			.value = CONFIG_CAVS_LPRO,
+		},
```

Comment:
far better to write
```
{EXT_MAN_CONFIG_CAVS_LPRO, CONFIG_CAVS_LPRO},
```
New elems can be quickly added.

Reply:
ok, changed